plugin/output/file/meta.go

```diff
@@ -0,0 +1,169 @@
+package file
```

Comment:
how about adding tests for meta operations?

Reply:
added into file_test.go